tests/test_handler.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+import os
+from datetime import datetime, timedelta
+
+from RadClass.analysis import RadClass
+from tests.create_file import create_file
+
+
+def test_integration():
```

Comment:
should probably move the code that sets up your file to another method

Reply:
I'm not sure what part specifically you are referencing, but I think everything defined in the test that is used to create the file using `create_file` must remain there for either running `RadClass` or comparing with expected further down.

Reply:
Ignore my comment - the more I look at it, the more I see this as a good design.